app/presenters/layouts/school_presenter.rb

```diff
@@ -6,12 +6,19 @@ def props
         school_logo_path: school_logo_path,
         school_icon_path: school_icon_path,
         courses: courses,
-        is_course_author: current_user_is_a_course_author?
+        is_course_author: current_user_is_a_course_author?,
+        has_notifications: notifications?
       }
     end
 
     private
 
+    def notifications?
+      return false if current_user.blank?
+
+      current_user.notifications.where(read_at: nil).any?
```

Comment:
@bodhish Reuse the model's scope, maybe?

```ruby
current_user.notifications.unread.any?
```

Reply:
Fixed in 15b24ff7c18d657c53f1d1f5eb9d08e834ee5d9d